src/components/Forms/ExclusiveContents/create/index.tsx

```diff
@@ -0,0 +1,77 @@
+import React, {useState} from 'react';
+import {View, Keyboard, ScrollView, Dimensions} from 'react-native';
+
+import {useTheme} from '../../../../hooks/useTheme';
+import {useModal} from '../../../../hooks/useModal';
+import {validateForm} from '../../../../utils/validators';
+import {ProjectAttrs} from './types';
+import ValidationFeedback from '../../../FormElements/ValidationFeedback';
+import {ButtonThemes} from '../../../Elements/Button/types';
+import {Button, Input} from '../../../Elements';
+import type {PostExclusiveContentsFormProps} from './types';
+import {schema} from './schema';
+import themedStyles from './styles';
+import PostExclusiveContentsReview from './Review';
+
+const PostExclusiveContentsForm = ({style, projectId}: PostExclusiveContentsFormProps) => {
+  const [data, setData] = useState<Partial<ProjectAttrs>>({
+    name: '',
```

Comment:
`title`, `project` (project ID), `accessible_tiers` (tier IDs), `public_access` (boolean), `description` (string) are correct values. We'll add `media` later.